model/ConversionRateSource.ts

```diff
@@ -0,0 +1,9 @@
+const conversionRateSources = ["STATIC", "BANK_OF_ENGLAND_DAILY", "EXCHANGE_RATES_IO"] as const
```

Comment:
tbh this could just be a string rather than an enum, as the value isnt important to the front end